packages/vite-plugin/src/node/fileWriter.ts

```diff
@@ -1,4 +1,5 @@
 import fsx from 'fs-extra'
+import path from 'path'
```

Comment:
@intellild Have you tested this on Windows? IIRC, we need to use `node:path/posix` here for Windows compatibility.